implementations/rust/ockam/ockam_app/tauri.conf.json

```diff
@@ -25,7 +25,7 @@
       "category": "DeveloperTool",
       "copyright": "",
       "deb": {
-        "depends": []
+        "desktopTemplate": "packaging/linux/application-description-template"
```

Comment:
It's almost certainly harmless to include now, but any chance this one snuck in from your other working branch?

Reply:
ops, yep, it snuck in, and it shouldn't be necessary anymore since I'm patching tauri directly